controller/api/models.py

```diff
@@ -251,9 +249,13 @@ def restart(self, **kwargs):
 
     def _clean_app_logs(self):
         """Delete application logs stored by the logger component"""
-        path = os.path.join(settings.DEIS_LOG_DIR, self.id + '.log')
-        if os.path.exists(path):
-            os.remove(path)
+        try:
+            url = 'http://{}:{}/{}/'.format(settings.LOGGER_HOST, settings.LOGGER_PORT, self.id)
+            requests.delete(url)
+        except Exception:
```

Comment:
Not sure I understand the rationale.  Any exception that occurs as that spot (regardless of what kind) is only a failure getting the logger to delete logs-- which should be eaten, imo, because that need not result in the app destroy request failing.  At most, I am thinking I should have at least logged the error.  Not doing so was an oversight that I should fix.